uid2_client/refresh_keys_util.py

```diff
@@ -0,0 +1,68 @@
+import datetime as dt
+import json
+from datetime import timezone
+
+from .keys import EncryptionKey, EncryptionKeysCollection
+from .request_response_util import *
+from .identity_scope import IdentityScope
+
+
+def _make_dt(timestamp):
+    return dt.datetime.fromtimestamp(timestamp, tz=timezone.utc)
+
+
+def parse_keys_json(resp_body):
+    keys = []
+    identity_scope = IdentityScope.UID2
+    if resp_body["identity_scope"] == "EUID":
+        identity_scope = IdentityScope.EUID
+    for key in resp_body["keys"]:
+        keyset_id = key.get("keyset_id")
```

Comment:
"keyset_id" is removed from key objects. Should we also remove it here?

https://atlassian.thetradedesk.com/confluence/display/UID2/UID2-1858+DSP+Check+-+Validate+30-day+token+is+not+in+bid+stream#UID21858DSPCheckValidate30daytokenisnotinbidstream-SDKs

Reply:
NVM, this function is also used in the deprecated client class.

Reply:
its being removed from `/key/bidstream` response but it will continue to exist in `/key/sharing` response